### PR TITLE
[Homework #19] - React Hooks by Ivan Ishchenko

### PR DESCRIPTION
**Pull request by ViVse**

### CODE REVIEW

File: homeworks/ivan-ishchenko/19-react-hooks/src/components/Input.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const Input = ({
+  className,
+  placeholder = '',
+  value,
+  onChange,
+  readonly = false,
+  cursor = 'inherit',
+  inputRef,
+}) => {
+  let props = {};
```
Comment (VitalikMelnyk) [inline homeworks/ivan-ishchenko/19-react-hooks/src/components/Input.js:12]: What is the goal of these two lines of code?
When you want to pass ref from parent to child, you need to use forwardRef.
https://react.dev/reference/react/forwardRef

File: homeworks/ivan-ishchenko/19-react-hooks/src/App.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+function App() {
+  const [tasks, setTasks] = useState([]);
+  const [newTask, setNewTask] = useState('');
+  const [didMount, setDidMount] = useState(false);
+
+  const changeNewTaskHandler = e => setNewTask(e.target.value);
+
+  const addTaskHandler = () => {
+    if (newTask.trim().length === 0) {
```
Comment (VitalikMelnyk) [inline homeworks/ivan-ishchenko/19-react-hooks/src/App.js:15]: Add validation for maximum input characters (for example 40).

**Review by VitalikMelnyk — CHANGES_REQUESTED** [review]
Hello @ViVse!
I have checked your PR and left some comments.

**Comment by ViVse** [comment]
@VitalikMelnyk, fixed all problems

**Review by VitalikMelnyk — APPROVED** [review]
Approved.

**Pull request merged**